> Built and tested successfully (via colima and docker/docker-compose) on `x86_64-darwin`.
> 
> But ofborg says `lima.passthru.tests on aarch64-darwin` failed:
> 
> ```
> > building
> > mkdir -p _output/bin
> > cp -a ./cmd/lima _output/bin/lima
> > # The hostagent must be compiled with CGO_ENABLED=1 so that net.LookupIP() in the DNS server
> > # calls the native resolver library and not the simplistic version in the Go library.
> > CGO_ENABLED=1 go build -ldflags="-s -w -X github.com/lima-vm/lima/pkg/version.Version=v0.14.1" -tags "no_vz" -o _output/bin/limactl ./cmd/limactl
> > codesign --entitlements vz.entitlements -s - ./_output/bin/limactl
> > libc++abi: terminating with uncaught exception of type std::runtime_error: file is already signed. pass -f to sign regardless.
> > make: *** [Makefile:182: codesign] Abort trap: 6
> ```

I have a little feeling that it may be because the binary was already built and not cleaned up or something like that. I added a `rm -rf _output` and will try again.

--- a/pkgs/applications/virtualization/lima/default.nix
+++ b/pkgs/applications/virtualization/lima/default.nix
@@ -27,8 +27,9 @@ buildGoModule rec {
 
   # clean fails with read only vendor dir
   postPatch = ''
-    substituteInPlace Makefile --replace 'binaries: clean' 'binaries:'
-    substituteInPlace Makefile --replace 'codesign --entitlements vz.entitlements -s -' 'codesign --force --entitlements vz.entitlements -s -'
+    substituteInPlace Makefile \
+      --replace 'binaries: clean' 'binaries:' \
+      --replace 'codesign --entitlements vz.entitlements -s -' 'codesign --force --entitlements vz.entitlements -s -'
   '';
 
   # It attaches entitlements with codesign and strip removes those,

--- a/pkgs/applications/virtualization/lima/default.nix
+++ b/pkgs/applications/virtualization/lima/default.nix
@@ -28,6 +28,7 @@ buildGoModule rec {
   # clean fails with read only vendor dir
   postPatch = ''
     substituteInPlace Makefile --replace 'binaries: clean' 'binaries:'
+    substituteInPlace Makefile --replace 'codesign --entitlements vz.entitlements -s -' 'codesign --force --entitlements vz.entitlements -s -'
   '';
 
   # It attaches entitlements with codesign and strip removes those,
@@ -36,7 +37,6 @@ buildGoModule rec {
 
   buildPhase = ''
     runHook preBuild
-    rm -rf _output
     make "VERSION=v${version}" binaries
     runHook postBuild
   '';

--- a/pkgs/applications/virtualization/lima/default.nix
+++ b/pkgs/applications/virtualization/lima/default.nix
@@ -36,6 +36,7 @@ buildGoModule rec {
 
   buildPhase = ''
     runHook preBuild
+    rm -rf _output
     make "VERSION=v${version}" binaries
     runHook postBuild
   '';

